scripts/migration/migrate_date_last_login_for_current_users.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import logging
+import datetime
+
+from website import models
+from website.app import init_app
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    init_app(routes=False)
+    dry_run = 'dry' in sys.argv
+    logger.warn('All active users will have their date_last_login update to now')
+    if dry_run:
+        logger.warn('Dry_run mode')
+    for user in models.User.find():
+        logger.info('User {0} "date_last_login" updated'.format(user.username))
+        if not dry_run and user.is_active:
+            user.date_last_login = datetime.datetime.utcnow()
+            user.save()
```

Comment:
Since we want logging when we update date_last_login (see previous comment), you can move the `if dry_run` check here, so that we only save if not in dry_run.